src/Worker.Extensions.DurableTask/FunctionsDurableClientProvider.netstandard.cs

```diff
@@ -10,7 +11,7 @@ namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask;
 
 internal partial class FunctionsDurableClientProvider
 {
-    private static Channel CreateChannel(ClientKey key, int? maxGrpcMessageSize)
+    private static Channel CreateChannel(ClientKey key, int? maxGrpcMessageSize, TimeSpan grpcHttpClientTimeout)
```

Comment:
when using Grpc.Core in .NET Standard, there isn't a default timeout enforced on gRPC requests. Unlike Grpc.Net.Client, which uses HttpClient and has a default timeout of 100 seconds  So basically we don't need to set any timeout here, we pass this argument only to align with the other one at NET 6.